src/reducers/index.js

```diff
@@ -72,8 +74,14 @@ function getInitialState() {
      * Output state passed as props to the the rendered FixedDataTable
      * NOTE (jordan) rows may contain undefineds if we don't need all the buffer positions
      */
+    columnOffsets: {},
+    fixedColumnOffsets: {},
+    fixedRightOffsets: {},
```

Comment:
Are fixedColumnOffsets & fixedRightOffsets used anywhere?  I don't see it in this review.

Reply:
nope, not used